requirements.txt

```
@@ -1 +1 @@
-sqlalchemy==1.3.17
+sqlalchemy
```

```suggestion
sqlalchemy==1.4.4
```

I'd prefer all `requirements.txt` entries stay pinned for stability of testing.